### DISCUSSION TITLE
fusionner les messages de trouvaille et remplacement des items

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#686] [Nouvelles idées]
Autant le faire dans un seul message, surtout que refaire la présentation permettra peut-être de faire en sorte que les gens se trompent moins par rapport à quel item est vendu

**Comment by BastLast** [comment]
pr info le faire dans 2 messages a justement été fait pr que les gens se trompent moins car les gens ne lisaient que le début de l'embed
